capture/genBitmaps.js

```diff
@@ -169,12 +170,48 @@ function capturePageSelectors(url,scenarios,viewports,bitmaps_reference,bitmaps_
 
           var filePath      = (isReference)?reference_FP:test_FP;
 
-
           if(o === "body:noclip" || o === "document") {
             casper.capture(filePath);
           } else if (casper.exists(o)) {
             if (casper.visible(o)) {
-              casper.captureSelector(filePath, o);
+ 
+                var targetElementsLength,
+                    newTestFiles = [];
+
+                // check if there are more elements on the same page for a given reference selector
+                targetElementsLength = casper.evaluate(function (o) {
+                    var allSimilarElements = document.querySelectorAll(o);
+                    [].forEach.call(allSimilarElements, function(element, index) {
+
+                        // let's add the temporary CSS class name to group the similar elements
+                        element.classList.add('backstopJS' + index); 
+                    });
+                    return allSimilarElements.length;
+                }, o);
+
+                // if there are more elements that has similar CSS seletors
+                if (targetElementsLength > 1) {
+                    var insertPos = fileName.split('_', 2).join('_').length,
```

Comment:
Sure. I will take a look at #197 and come back, so that the discussion would make sense.